solarforecastarbiter/io/reference_observations/default_forecasts.py

```diff
@@ -112,5 +112,71 @@ def is_in_nwp_domain(site):
 ]
 
 
+TEMPLATE_DETERMINISTIC_PERSISTENCE_FORECASTS = [
+    Forecast(
+        name='Day Ahead Persistence',
+        issue_time_of_day=dt.time(0),
+        lead_time_to_start=pd.Timedelta('24h'),
+        interval_length=pd.Timedelta('1h'),
+        run_length=pd.Timedelta('1h'),
+        interval_label='ending',
+        interval_value_type='interval_mean',
+        variable='ghi',
+        site=_DUMMY_SITE,
+        extra_parameters=json.dumps(
+            {'is_reference_persistence_forecast': True,
+             'index_persistence': False})
+        ),
+    Forecast(
+        name='Hour Ahead Persistence',
+        issue_time_of_day=dt.time(0),
+        lead_time_to_start=pd.Timedelta('1h'),
+        interval_length=pd.Timedelta('1h'),
+        run_length=pd.Timedelta('1h'),
+        interval_label='beginning',
+        interval_value_type='interval_mean',
+        variable='ghi',
+        site=_DUMMY_SITE,
+        extra_parameters=json.dumps(
+            {'is_reference_persistence_forecast': True,
+             'index_persistence': False})
```

Comment:
I should have noticed this the first time through... why default to index=False? I think index=True would be more interesting and useful to most users.

Reply:
I'd be ok with both but I'm guessing you'd rather pick one.

Reply:
I didn't want to fuss with adjusting it to true/false based on the variable since  only works for power/irradiance. It would actually be easy to do though